SSA/Projects/FullyHomomorphicEncryption/Statements.lean

```diff
@@ -0,0 +1,234 @@
+import SSA.Projects.FullyHomomorphicEncryption.Basic
+import Std.Data.List.Lemmas
+import Mathlib.Data.List.Basic
+
+variable {q t : Nat} [ hqgt1 : Fact (q > 1)] {n : Nat}
+variable (a b : R q n)
+
+theorem poly_mul_comm : a * b = b * a := by
```

Comment:
I think it would make more sense to put these definition in a namespace, e.g. `Polynomial`, and call them `Polynomial.mul_comm`